sunpy/timeseries/sources/eve.py

```diff
@@ -57,32 +57,48 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, title="EVE/ESP Level 1", **kwargs):
```

Comment:
On the axes in plot, the original plot method on time series base takes an axes and I think each plot method should as a result at least keep that as a named argument. The idea is to keep the API uniform, we can add title after the fact but I am also leaning toward removing title as an arg fully.

Thoughts?

Reply:
I guess that makes sense but the user should pass the axes from a pandas dataframe, else this won't really behave the same way it does when `ax` from `fig, ax = plt.subplots()` is passed to it directly, and `plt.gca` won't work either

Reply:
The issue is that the plots created have multiple axes, so it's not possible to have a single `axes` argument here. I would remove the title argument though, and just always set the title to be what is currently the default. It's easy enough for the user to override this themselves after calling `.plot()`.